deploy/podman/README.md

```diff
@@ -1,24 +1,74 @@
-# Deploy with podman
+# Deploy with Podman
 
 In this scenario the service and associated components are deployed onto your
 local host in containers using [Podman](https://podman.io/).
 
-## Requirements
+## General Requirements
 
-Make sure you have [podman](https://podman.io) version 3.3+ installed. If you
+Make sure you have [Podman](https://podman.io) version 3.3+ installed. If you
 must use an older version of podman, reference the [previous documentation and
 procedure](https://github.com/openshift/assisted-service/tree/v2.0.11#deploy-without-a-kubernetes-cluster)
 to avoid a [podman bug](https://github.com/containers/podman/issues/9609).
 
-Grab [pod.yml](./pod.yml) and [configmap.yml](configmap.yml) from this
-directory. No need to clone the whole repo.
+## Requirements for an HTTP based deployment
+
+### Files
+
+No need to clone the whole repo. Use:
+* [pod.yml](./pod.yml)
+* [configmap.yml](configmap.yml)
+
+### Environment Variables
 
 Change `IMAGE_SERVICE_BASE_URL` and `SERVICE_BASE_URL` in
 [configmap.yml](./configmap.yml) to match the hostname or IP address of your
 host. For example if your IP address is 192.168.122.2, then the
 `SERVICE_BASE_URL` would be set to <http://192.168.122.2:8090>. Port 8090 is
 the assisted-service API that agents will connect to.
 
+## Requirements for an HTTPS based deployment
```

Comment:
It's worth a sentence or two about why to do it this way. It would especially be valuable to clarify that the certs will be automatically added to generated ISOs, and thus communication between agents and the service will be secured (I assume that's all correct?).

Reply:
Yes. I think it's worth stating directly which communication paths will be secured by deploying this way.

Reply:
Added.